web/src/features/Gitops/context.js

```diff
@@ -160,15 +160,7 @@ const GitOpsProvider = ({ children }) => {
     setSelectedService(service);
   };
 
-  const getGitOpsInput = (
-    uri,
-    tempBranch,
-    tempPath,
-    format,
-    action,
-    newHttpPort,
-    newSshPort
```

Comment:
got unused var error with these. LMK if i should add these back

Reply:
<img width="874" alt="Screen Shot 2022-09-08 at 15 11 34" src="https://user-images.githubusercontent.com/4998130/189218623-d97c66a5-643c-4159-923e-0f38b5e1255c.png">
I used this to verify that the build-web job will fail when there are linting errors now.

Reply:
@miaawong ^

Reply:
Perfect! 💯